app/src/main/res/values/styles.xml

```diff
@@ -24,6 +24,15 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="radio_button_custom_style" parent="Widget.AppCompat.CompoundButton.RadioButton">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:foreground">?attr/selectableItemBackgroundBorderless</item>
+        <item name="android:paddingStart">16dp</item>
+        <item name="android:paddingEnd">16dp</item>
```

Comment:
`android:paddingStart` and `android:paddingEnd` could be called  from a dimen resource.